test/endpoint_tests/test_graphql.py

```diff
@@ -0,0 +1,153 @@
+from boxwise_flask.db import db
+from boxwise_flask.models import Camps, Cms_Users
+
+
+def test_all_bases(client):
+    """example test for graphql endpoint"""
+
+    db.connect_db()
+    Camps.create(id=1, organisation_id=1, name="some text1", currencyname="hello")
+    Camps.create(id=2, organisation_id=1, name="some text1", currencyname="hello")
+    Camps.create(id=3, organisation_id=1, name="some text1", currencyname="hello")
+    Camps.create(id=4, organisation_id=1, name="some text1", currencyname="hello")
+    Camps.create(id=5, organisation_id=1, name="some text1", currencyname="hello")
+    db.close_db(None)
+
+    graph_ql_query_string = """query { \
+                allBases { \
+                    id \
+                    organisation_id \
+                    name \
+                } \
+            }"""
+
+    data = {"query": graph_ql_query_string}
+    response_data = client.post("/graphql", json=data)
+    assert response_data.status_code == 200
+    assert response_data.json["data"]["allBases"][0]["id"] == 1
+
+
+def test_base(client):
+    """example test for graphql endpoint"""
+
+    db.connect_db()
+    Camps.create(id=1, organisation_id=1, name="some text1", currencyname="hello")
+    Camps.create(id=2, organisation_id=1, name="some text1", currencyname="hello")
+    Camps.create(id=3, organisation_id=1, name="some text1", currencyname="hello")
+    Camps.create(id=4, organisation_id=1, name="some text1", currencyname="hello")
+    Camps.create(id=5, organisation_id=1, name="some text1", currencyname="hello")
+    db.close_db(None)
+
+    graph_ql_query_string = """query Base {
+                base(id: "1") {
+                    id
+                    organisation_id
+                    name
+                    currencyname
+                }
+            }"""
+    data = {"query": graph_ql_query_string}
+    response_data = client.post("/graphql", json=data)
+    assert response_data.status_code == 200
+    assert response_data.json["data"]["base"]["id"] == 1
+
+
+def test_all_users(client):
+    """example test for graphql endpoint"""
+
+    db.connect_db()
+    Cms_Users.create(
```

Comment:
If only the ids and emails are relevant, you could create a list of emails first and then run
```
for i, email in enumerate(emails):
     Cms_Users.create(id=i, email=email, ...)
```
which is more concise